### ISSUE TITLE
[CI] DebMetadataTests test05CheckLintian failing

### ISSUE THREAD

**Issue by stu-elastic** [elastic/elasticsearch#100068]
The failure is 
```
org.elasticsearch.packaging.util.Shell$ShellException: Command was not successful: [bash -c lintian --fail-on warning --suppress-tags malformed-override,mismatched-override /var/lib/jenkins/workspace/elastic+elasticsearch+main+multijob+packaging-tests-unix/os/debian-11-packaging/distribution/packages/deb/build/distributions/elasticsearch-8.11.0-SNAPSHOT-amd64.deb]	
   result: exitCode = [2] stdout = [W: elasticsearch: non-standard-dir-perm usr/share/elasticsearch/plugins/ 2755 != 0755
```

**Build scan:**
https://gradle-enterprise.elastic.co/s/erueqqw2zr3wg/tests/:qa:os:destructiveDistroTest.default-deb/org.elasticsearch.packaging.test.DebMetadataTests/test05CheckLintian

**Reproduction line:**
```
null
```

**Applicable branches:**
main

**Reproduces locally?:**
No

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.packaging.test.DebMetadataTests&tests.test=test05CheckLintian

**Failure excerpt:**
```
org.elasticsearch.packaging.util.Shell$ShellException: Command was not successful: [bash -c lintian --fail-on warning --suppress-tags malformed-override,mismatched-override /var/lib/jenkins/workspace/elastic+elasticsearch+main+multijob+packaging-tests-unix/os/debian-11-packaging/distribution/packages/deb/build/distributions/elasticsearch-8.11.0-SNAPSHOT-amd64.deb]
   result: exitCode = [2] stdout = [W: elasticsearch: non-standard-dir-perm usr/share/elasticsearch/plugins/ 2755 != 0755
N: 404 hints overridden (194 errors, 158 warnings, 52 info); 6 unused overrides] stderr = [warning: running with root privileges is not recommended!]

  at __randomizedtesting.SeedInfo.seed([59F809A419F47149:99069548A38014F2]:0)
  at org.elasticsearch.packaging.util.Shell.runScript(Shell.java:143)
  at org.elasticsearch.packaging.util.Shell.run(Shell.java:73)
  at org.elasticsearch.packaging.test.DebMetadataTests.test05CheckLintian(DebMetadataTests.java:45)
  at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
  at java.lang.reflect.Method.invoke(Method.java:580)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
  at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:1583)

```

**Comment by stu-elastic** [comment]
We are seeing this on debian-11 packaging (above), [ubuntu 18.04](https://gradle-enterprise.elastic.co/s/tnhsxxzsp4bsi) and [ubuntu 22.04](https://gradle-enterprise.elastic.co/s/m5mlhppwu5b2g).

**Comment by breskeby** [comment]
This should just be fixed by https://github.com/elastic/elasticsearch/pull/100063

**Comment by breskeby** [comment]
I keep an eye on package tests on main to double check we can consider this fixed

**Comment by mark-vieira** [comment]
@breskeby this still looks to be failing.

**Comment by breskeby** [comment]
Damn it. running https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+pull-request+packaging-tests-unix-sample/ wasn't enough to validate the fix. I messed that up. Turns out the fix we did for the nebula plugin wasn't complete as `setgid` handling is still broken for copy spec specific permission. I worked on a patch for the nebula plugin and have raised https://github.com/elastic/elasticsearch/pull/100275 that should unblock us for now by using this patched version. I'll raise a PR to nebula but this should unblock us for now.
